### PR TITLE
Draft of #28005 -- Normalized usage of terms 'argument' and 'parameter' 

### PR DESCRIPTION
**Pull request by lachilles**
Normalized usage of terms 'argument' and 'parameter' on docs

### CODE REVIEW

File: docs/releases/1.11.txt [hunk 1]
```diff
@@ -202,7 +202,7 @@ Minor features
 :mod:`django.contrib.postgres`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* The new ``distinct`` argument for
+* The new ``distinct`` parameter for
```
Comment (lachilles) [inline docs/releases/1.11.txt]: Refers to https://github.com/django/django/blob/487362fa8f23d41de4db58e7408e66eb36399af0/django/contrib/postgres/aggregates/general.py#L13

File: docs/releases/2.0.txt [hunk 2]
```diff
@@ -104,7 +104,7 @@ Minor features
 :mod:`django.contrib.postgres`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* The new ``distinct`` argument for
+* The new ``distinct`` parameter for
```
Comment (lachilles) [inline docs/releases/2.0.txt]: Refers to https://github.com/django/django/blob/487362fa8f23d41de4db58e7408e66eb36399af0/django/contrib/postgres/aggregates/general.py#L13

File: docs/releases/1.0-porting-guide.txt [hunk 3]
```diff
@@ -41,7 +41,7 @@ Common changes to your models file:
 Rename ``maxlength`` to ``max_length``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Rename your ``maxlength`` argument to ``max_length`` (this was changed to be
+Rename your ``maxlength`` parameter to ``max_length`` (this was changed to be
```
Comment (lachilles) [inline docs/releases/1.0-porting-guide.txt]: Refers to max_length parameter seen in various model.py files

File: docs/releases/1.1.txt [hunk 4]
```diff
@@ -141,7 +141,7 @@ and expected.
 Permanent redirects and the ``redirect_to()`` generic view
 ----------------------------------------------------------
 
-Django 1.1 adds a ``permanent`` argument to the
+Django 1.1 adds a ``permanent`` parameter to the
```
Comment (lachilles) [inline docs/releases/1.1.txt]: Refers to parameter below: https://github.com/lachilles/django/blob/301de774c21d055e9e5a7073e5bffdb52bc71079/django/shortcuts.py#L40-L40

File: docs/releases/1.1.txt [hunk 5]
```diff
@@ -288,7 +288,7 @@ A couple of small -- but highly useful -- improvements have been made to the
 test client:
 
 * The test :class:`Client` now can automatically follow redirects with the
-  ``follow`` argument to :meth:`Client.get` and :meth:`Client.post`. This
+  ``follow`` parameter to :meth:`Client.get` and :meth:`Client.post`. This
```
Comment (lachilles) [inline docs/releases/1.1.txt]: Refers to parameter below:
https://github.com/lachilles/django/blob/3db84b5526a16a136862381c6f8b08c59da4cb82/django/test/client.py#L514-L514

File: docs/releases/1.10.txt [hunk 6]
```diff
@@ -122,12 +122,12 @@ Minor features
   to prevent an issue where Safari caches redirects and prevents a user from
   being able to log out.
 
-* Added the optional ``backend`` argument to :func:`~django.contrib.auth.login`
+* Added the optional ``backend`` parameter to :func:`~django.contrib.auth.login`
```
Comment (lachilles) [inline docs/releases/1.10.txt]: Refers to backend parameter:
https://github.com/lachilles/django/blob/a3ba2662cdaa36183fdfb8a26dfa157e26fca76a/django/contrib/auth/__init__.py#L118-L118

File: docs/releases/1.10.txt [hunk 7]
```diff
@@ -122,12 +122,12 @@ Minor features
...
   to allow using it without credentials.
 
 * The new :setting:`LOGOUT_REDIRECT_URL` setting controls the redirect of the
   :func:`~django.contrib.auth.views.logout` view, if the view doesn't get a
-  ``next_page`` argument.
+  ``next_page`` parameter.
```
Comment (lachilles) [inline docs/releases/1.10.txt]: Refers to next_page parameter
https://github.com/lachilles/django/blob/e7dc39fb65e51d7613c941f7e5768b621dea4e76/django/contrib/auth/views.py#L198-L198

**Comment by levidyrek** [comment]
Should be closed in favor of #9238.

**Pull request closed without merging**